src/main/java/org/jenkinsci/plugins/workflow/cps/DSL.java

```diff
@@ -98,6 +104,14 @@ protected Object readResolve() throws IOException {
         return this;
     }
 
+    /** Allows user to disable saving step info, if this imposes too high a performance overhead
+     *   by setting System Property org.jenkinsci.plugins.workflow.cps.DSL.keepStepInfo to 'false'.
+     *  This may be modified at runtime by the user if needed for debugging. */
+    private static boolean isKeepStepInfo() {
+        String prop = System.getProperty(DSL.class.getName()+".keepStepInfo");
+        return (StringUtils.isEmpty(prop)|| Boolean.parseBoolean(prop));
```

Comment:
:ant: `return Boolean.parseBoolean(prop, "true");` would be simpler and achieve the same result, right?